studying-c-sharp-mark-kotlobay/static_demo/Program.cs

```diff
@@ -0,0 +1,9 @@
+﻿using studying_c_sharp_mark_kotlobay.static_demo;
+
+internal class Program
```

Comment:
You don't need to move this file into another folder, since this is the entry point of the program and it's logic to be on the root folder. 

Also, change private, internal, into public.

Reply:
Thank you for comment. Done updating